### ISSUE TITLE
ProgressBarAndroid Component does not announce Checked/Unchecked State

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30963]
## Description
Does not announce "checked"/"not checked" on focus of ProgressBarAndroid Component.

This issue is covered in-depth by parent issue #30843, please check there for more details.

## React Native version:
0.63

**Comment by smash-96** [comment]
ProgressBarAndroid package has been deprecated. 
Should we focus on this issue?

**Comment by safaiyeh** [comment]
I believe this was moved to a community repo here: https://github.com/react-native-progress-view/progress-view

**Comment by blavalla** [comment]
While technically possible, it's not really valid to have a progress bar that is checkable, so closing out.
